LTC_extracts_DRAFT.R

```diff
@@ -0,0 +1,103 @@
+
+#####################################################
+# Draft LTC Extract Processing Code
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description - Takes IT extract of LTC data.
+#              Renames and checks types of data.
+#              Flags variables which fall outside FY.
+#              Saves outputted flagged dataset.
+#####################################################
+
+## packages ##
+
+library(dplyr)
+library(stringr)
+library(readr)
+
+## financial year in question ##
+FY <- 1718
+year <- convert_fyyear_to_year(FY)
+end_fy <- lubridate::dmy(paste0("01-04-", as.numeric(substr(year, 3, 4)) + 1))
```

Comment:
We don't need lubridate here, and `dmy` is usually slower because it's 'clever'. We can just specify the date in a more standard format(i.e. YYYY-MM-DD)

```suggestion
FY <- 1718
end_fy <- as.Date(paste0(convert_fyyear_to_year(FY), "-03-31")
```

Reply:
lubricate removed

added +1 to first part of the date as we are looking at the end of fy year so should be "2017 + 1"? please check my logic is correct @Moohan

Reply:
Yep you're correct, that looks good now